tests/coding_guidelines/c99_cplusplus98_test.sh

```diff
@@ -24,8 +24,7 @@ INCLUDE_FILES=`ls $INCLUDE_DIR/openssl/*.h | grep -v $INCLUDE_DIR/openssl/arm_ar
 # some non-ISO practices, but not all — only those for which ISO C requires a
 # diagnostic, and some others for which diagnostics have been added."
 # https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html
-
-${CC} -std=c99 -c -I${INCLUDE_DIR} -include ${INCLUDE_FILES} -Wpedantic -fsyntax-only -Werror
+${CC} -std=c99 -c -I${INCLUDE_DIR} $(echo ${INCLUDE_FILES} | sed 's/[^ ]* */-include &/g') -Wpedantic -fsyntax-only -Werror ./tests/compiler_features_tests/builtin_swap_check.c
```

Comment:
What is this sed removing from the list? And why add the builtin_swap_check to all of these targets?

Reply:
sed's mainly in charge of prepending an `-include` to each file in `$INCLUDE_FILES`. What we had previously didn't actually work :(. This also needs a basic "c" file to compile, so I chose `builtin_swap_check` since that was a good existing example that works with older compilers